### ISSUE TITLE
Improve Prometheus Service Configurator robustness against input files

### ISSUE THREAD

**Issue by oleg-nenashev** [keptn-contrib/prometheus-service#228]
See https://github.com/keptn/keptn.github.io/issues/984 I hit when passing through the quickstart. While I applied a fix in https://github.com/keptn/examples/pull/215, I believe that the service should be more robust against such configurations when `< 10` used as a SLI expression. It is a totally valid syntax from user standpoint, and Keptn does not provide any verification that would help to discover the issue early

**Comment by oleg-nenashev** [comment]
As @christian-kreuzberger-dtx commented, it is likely to be a Keptn core issue. I will create one. At the same time it would be nice to add some diagnostics to the prometheus service

**Comment by christian-kreuzberger-dtx** [comment]
Hi,

when I read the details in Slack you were always talking about SLIs, but in fact, this is about SLOs. 

While for SLI parsing we make use of keptn/go-utils, SLO parsing seems to be done directly in prometheus-service, so there is a good possibility to fix this in prometheus-service.

**Comment by christian-kreuzberger-dtx** [comment]
The following SLO entry will cause a problem:
```yaml
spec_version: '0.1.0'
comparison:
  compare_with: "single_result"
  include_result_with_score: "pass"
  aggregate_function: avg
objectives:
  - sli: failing_request
    displayName: "Failing Requests"
    pass:
      - criteria:
          - "< 10"
```

Please note the space for criteria, e.g., **< 10**

respective SLI file
```yaml
spec_version: '1.0'
indicators:
  failing_request: promhttp_metric_handler_requests_total{code!="200",job="$SERVICE-$PROJECT-$STAGE"}[$DURATION_SECONDS]
```

**Comment by christian-kreuzberger-dtx** [comment]
I have triaged the problem, and the "workaround" is to use a properly formated SLO.yaml.

My fix provided in PR #233 does not work, and I don't have time to dig deeper. I will leave this issue open for now.
